compiler/optimizer/OMRSimplifierHandlers.cpp

```diff
@@ -13256,6 +13406,11 @@ TR::Node *iflcmpgeSimplifier(TR::Node * node, TR::Block * block, TR::Simplifier
       if (firstChild->getOpCode().isLoadConst() && conditionalBranchFold((((uint64_t)originalFirst->getLongInt())>=((uint64_t)originalSecond->getLongInt())), node, firstChild, secondChild, block, s))
          return node;
       }
+
+   IfcmpgeReducer ifcmpReducer(node, s);
+   if (ifcmpReducer.reducible())
+      return ifcmpReducer.reduce();
```

Comment:
same comment: do we have to return here?

Reply:
Fixed.